src/datalad_installer.py

```diff
@@ -24,11 +24,17 @@
 from collections.abc import Callable, Iterator
 from contextlib import contextmanager, suppress
 import ctypes
+from dataclasses import InitVar, dataclass, field
+from email import policy
+from email.headerregistry import ContentTypeHeader
 from enum import Enum
 from functools import total_ordering
 from getopt import GetoptError, getopt
+from html.parser import HTMLParser
```

Comment:
note to myself: this is a standard Python library - https://python.readthedocs.io/en/v2.7.2/library/htmlparser.html (new in 2.2!) so we remain true to the goal here of no external depends! cool